forc-plugins/forc-doc/src/tests/data/impl_traits_generic/Forc.toml

```diff
@@ -3,6 +3,7 @@ authors = ["Fuel Labs <contact@fuel.sh>"]
 entry = "lib.sw"
 license = "Apache-2.0"
 name = "impl_traits_generic"
+implicit-std = false
 
 [dependencies]
-core = { path = "../../../../../../sway-lib-core" }
+ops = { path = "../ops" }
```

Comment:
Why `ops` here and not `std`?

Reply:
previously the test used core as it wasnt that large, so the hardcoded values in the test werent that long. but with the new std its comically long and unnecessary.